homeassistant/components/icloud/account.py

```diff
@@ -354,6 +354,11 @@ def fetch_interval(self) -> int:
         """Return the account fetch interval."""
         return self._fetch_interval
 
+    @property
```

Comment:
I compared with the Google Maps device tracker, and in that component a public variable is used instead of a getter method. I think for consistency sake, we should use that here too.

Reply:
I was wondering what the best practice was here.
Will do that! Thanks!